packages/config/config/devices/0x0001/zrw230.json

```diff
@@ -18,176 +18,44 @@
 	},
 	"associations": {
 		"1": {
-			"label": "Tapping paddle once",
-			"maxNodes": 5,
-			"isLifeline": true
+			"label": "Single Tap",
+			"maxNodes": 5
 		},
 		"2": {
-			"label": "Tapping paddle twice",
-			"maxNodes": 5,
-			"isLifeline": true
```

Comment:
Do these commands reach the controller without the groups marked as lifelines?

Reply:
I'm not actually sure. I'll re-add them.